snmp/osupdate

```diff
@@ -39,7 +40,7 @@ if command -v zypper &>/dev/null ; then
     if [ "$UPDATES" -ge 2 ]; then
         echo $(($UPDATES-2));
     else
-        echo "0";
+        echo "0,0";
```

Comment:
if a "section" does not support this feature ,
return 0 is a bad practice. "Null" is recommended